### ISSUE TITLE
Connection failure in Events view during Master Upgrade

### ISSUE THREAD

**Issue by DanTulovsky** [aptakube/aptakube#143]
### Current and expected behavior

Hi,

I kicked off a cluster master upgrade in a GKE regional cluster (3 masters).  While I am still able to access other views, the `Events` view shows me:

```
422: Unprocessable Entity
ListOptions.meta.k8s.io "" is invalid: resourceVersionMatch: Forbidden: resourceVersionMatch is forbidden when continue is provided
```

![image](https://github.com/aptakube/aptakube/assets/509887/fc6d4c3f-1d19-460f-b9dc-e9398a7d1a08)

I can still access events via the cloud.google.com ui and via `kubectl get events`.  So perhaps this is a bug.

Thank you
Dan

### Environment & Operating System

Kubernetes 1.24 being upgraded to 1.25.
Aptakube 1.5.3 on macOS

**Comment by goenning** [comment]
Thanks for reporting that, seems related to recent paging changes. Interesting that it only happens on `Events`. 
Tracking this upstream https://github.com/kube-rs/kube/issues/1278

**Comment by goenning** [comment]
This is fixed on 1.5.4. The paging changes were reverted until the upstream PR is merged.
